src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -0,0 +1,186 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import FieldHelpStyle from '../field-help/field-help.style';
+import HiddenCheckboxStyle from './hidden-checkbox.style';
+import LabelStyle from '../label/label.style';
+import StyledHelp from '../help/help.style';
+import checkBoxClassicStyle from './checkbox-classic.style';
+import { THEMES } from '../../../style/themes';
+
+const StyledCheckbox = styled.div`
+  ${({
+    checked, disabled, error, fieldHelpInline, inputWidth, reverse, size, theme
+  }) => css`
+    padding-top: 8px;
+
+    .carbon-checkbox__input {
+      padding-top: 1px;
+    }
+
+    svg {
+      background-color: ${theme.colors.white};
+      border: solid 1px ${theme.colors.border};
+    }
+
+    ${HiddenCheckboxStyle},
+    svg {
+      height: 16px;
+      position: absolute;
+      padding: 1px;
+    }
+
+    .carbon-checkbox__input,
+    ${HiddenCheckboxStyle},
+    svg {
+      box-sizing: border-box;
+      width: 16px;
+    }
+
+    ${HiddenCheckboxStyle}:not([disabled]) {
+      &:focus + svg,
+      &:hover + svg {
+        outline: solid 3px ${theme.colors.focus};
+      }
+    }
+
+    ${LabelStyle} {
+      padding: 0 6px;
+      text-align: ${({ labelAlign }) => labelAlign};
+      width: auto;
+
+      & ${StyledHelp} {
+        color: ${theme.help.color};
+        vertical-align: bottom;
+
+        &:hover, &:focus {
+          color: ${theme.text.color};
+        }
+      }
+    }
+
+    label:hover + .carbon-checkbox__input ${HiddenCheckboxStyle}:not([disabled]) + svg {
+      outline: solid 3px ${theme.colors.focus};
+    }
+
+    ${FieldHelpStyle} {
+      margin-left: 16px;
+      margin-top: 0;
+      padding-left: 6px;
+    }
+
+    ${() => theme.name !== THEMES.classic && size === 'large' && css`
```

Comment:
I think that should be just placed here without checking for the `classic` theme and then overridden in the classic style file

Reply:
Updated 👍